libraries/r-tensorflow-1.12/Dockerfile

```diff
@@ -0,0 +1,11 @@
+RUN echo ". /opt/conda/etc/profile.d/conda.sh" >> /etc/environment && \
+    echo "conda activate base" >> /etc/environment
+
+ENV PATH /opt/conda/bin:$PATH
+
+RUN conda install python=3.6 && \
+/opt/conda/bin/pip install tensorflow==1.12 && \
```

Comment:
nit: i like what you did above by lining up the commands that are in the same RUN command